PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -681,6 +681,15 @@ INT_PTR CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 	{
 		case WM_INITDIALOG :
 		{
+			HWND hFindCombo = ::GetDlgItem(_hSelf, IDFINDWHAT);
+			HWND hReplaceCombo = ::GetDlgItem(_hSelf, IDREPLACEWITH);
+			HWND hFiltersCombo = ::GetDlgItem(_hSelf, IDD_FINDINFILES_FILTERS_COMBO);
+
+			HFONT hf = CreateFont(-12, 0, 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, _T("Courier New"));

```

Comment:
True that.

Accidently made change to a new pull request:

https://github.com/notepad-plus-plus/notepad-plus-plus/pull/3361